recipes-support/swupdate/swupdate_%.bbappend

```diff
@@ -4,9 +4,11 @@ PACKAGECONFIG_CONFARGS = ""
 
 SRC_URI += " \
      file://swupdate.cfg \
+     file://swupdate-extra-args \
 "
 
 do_install_append() {
-    install -d ${D}${sysconfdir}
+    install -d ${D}${sysconfdir}/swupdate/conf.d
     install -m 644 ${WORKDIR}/swupdate.cfg ${D}${sysconfdir}
+    install -m 755 ${WORKDIR}/swupdate-extra-args ${D}${sysconfdir}/swupdate/conf.d/
```

Comment:
I prefer something like `<digit>-swupdate-extra-args`, you can check as an exemple
the following implementation:
 - https://github.com/sbabic/meta-swupdate-boards/blob/master/recipes-support/swupdate/swupdate_%25.bbappend#L13